validator/client/src/main/java/tech/pegasys/teku/validator/client/slashingriskactions/SlashedValidatorShutDown.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Consensys Software Inc., 2023
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.validator.client.slashingriskactions;
+
+import static tech.pegasys.teku.infrastructure.logging.StatusLogger.STATUS_LOG;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import tech.pegasys.teku.bls.BLSPublicKey;
+
+public class SlashedValidatorShutDown implements SlashingRiskDetectionAction {
+
+  @Override
+  public void perform(List<BLSPublicKey> pubKeys) {
+    if (!pubKeys.isEmpty()) {
+      STATUS_LOG.validatorSlashedAlert(
+          pubKeys.stream().map(BLSPublicKey::toAbbreviatedString).collect(Collectors.toSet()));
+      System.exit(2);
```

Comment:
- Split out the exit from the perform, and have it on the interface as something like
 ```
default void die() {
   System.exit(2);
}
```
- once that is split out, we can actually test this function and make sure the log happens
- we can also make sure the keys are displayed in a useful way. in this instance, i would argue print the full key not abbreviated key. This will be our best point to see the error message is correct when things get refactored in future.

Reply:
Done:
- Refactored the `SlashedValidatorAlert` to made it extend the `SlashedValidatorShutDown` and override the `shutDown` method to test the actual `SlashedValidatorShutDown`
- Print the hex representation of the public keys instead of the abbreviation

Reply:
last part of that was test?

Reply:
There is a test for ValidatorTimingActions where this class is tested. Check [7916](https://github.com/Consensys/teku/pull/7916/files#diff-a37ab63dcebb7b505f1e7805107f46b669deb78544a395726a33b66de4c92268)